master/internal/resourcemanagers/kubernetes/spec.go

```diff
@@ -365,7 +365,21 @@ func (p *pod) createPodSpec(ctx *actor.Context, scheduler string) error {
 		p.ports = append(p.ports, port)
 	}
 
-	envVars, err := p.configureEnvVars(spec.EnvVars(), env, deviceType)
+	specEnvVars := spec.EnvVars()
+	var specEnvFrom []k8sV1.EnvFromSource
+	podSpec := env.PodSpec()
+	if podSpec != nil {
+		for _, container := range podSpec.Spec.Containers {
+			if container.Name == "determined-container" {
```

Comment:
is this a const somewhere? seems like it should be.

Reply:
oops - forgot to change this